product-service/src/database/products.ts

```diff
@@ -0,0 +1,22 @@
+import productList from './productList.json';
+
+export interface IProduct {
+    count: number,
+    description: string,
+    id: string,
+    price: number,
+    title: string
+};
+
+const getAllProducts = async (): Promise<IProduct[]> => {
+    try {
+        const productsJson = productList;
+        console.log(productsJson);
+        return productsJson;
+    } catch (e) {
```

Comment:
I think it is worth to think about returning proper response with message something like `Failed to get products`